src/lightspeed_evaluation/core/llm/custom.py

```diff
@@ -128,5 +148,16 @@ def call(
 
             return results
 
+        except InternalServerError as e:
+            # Check if it's an SSL/certificate error
+            error_msg = str(e)
+            if "[X509]" in error_msg or "PEM lib" in error_msg:
+                logger.error("SSL certificate verification failed: %s", error_msg)
+                logger.error("Please check your SSL configuration.")
+                sys.exit(1)
```

Comment:
If we really want to crash here (do we?) I suggest re-raising the exception:

```suggestion
                raise e
```